### PR TITLE
[MB-15057] Include actual delivery date in payload

### PR DESCRIPTION
**Pull request by ruizajtruss**
## Summary

Service counselors need to be able to see the `actual delivery date` for mto shipments. The column was not previously included in the payload, but now it is 🥳 

## Setup to Run Your Code

<details>
<summary>💻 You will need to use three separate terminals to test this locally.</summary>

##### Terminal 1

Start the Storybook locally.

```sh
make storybook
```

##### Terminal 2

Start the UI locally.

```sh
make client_run
```

##### Terminal 3

Start the Go server locally.

```sh
make server_run
```

</details>

### Additional steps

1. Login as a services counselor
2. Select a move (`I selected Taylor III, Edgar - move code: #K7HWM9`)
3. Type in a date value into the `actual_delivery_date` column in the `mto_shipments` table for your selected move (If you selected the move mentioned above, you can find it under `id: "3d248abd-0565-4ec8-b54f-0cd48322946d"`)
4. Navigate to the network tab and select `mto_shipments`
5. The new field should be present 🎉 

## Verification Steps for Author

These are to be checked by the author.
- [x] Have the Jira acceptance criteria been met for this change?

## Verification Steps for Reviewers

These are to be checked by a reviewer.

### Backend

- [ ] Code follows the guidelines for [Logging](https://transcom.github.io/mymove-docs/docs/dev/contributing/backend/Backend-Programming-Guide#logging)
- [ ] The requirements listed in [Querying the Database Safely](https://transcom.github.io/mymove-docs/docs/dev/contributing/backend/Backend-Programming-Guide/#querying-the-database-safely) have been satisfied.

#### Any new migrations/schema changes:

- [ ] Follows our guidelines for [Zero-Downtime Deploys](https://transcom.github.io/mymove-docs/docs/dev/contributing/database/Database-Migrations#zero-downtime-migrations)
- [ ] Have been communicated to #g-database
- [ ] Secure migrations have been tested following the instructions in our [docs](https://transcom.github.io/mymove-docs/docs/dev/contributing/database/Database-Migrations#secure-migrations)

## Screenshots
Before: 
image

After 🎉 :
image (4)

### CODE REVIEW

**Comment by rogeruiz** [comment]
hi @ruizajtruss, could you rebase your work on top of `main` or merge `main`
into your branch so it includes the new Go version 1.20.1 from #10185 rather
than the current version on this branch 1.19.3. Thanks. This will help me in
reviewing this and testing it locally.

**Comment by rogeruiz** [comment]
I'm doing the merge locally too to see if any tests break. So if this passes
locally, you won't need to actually do the merge and I'll give my review. 🥳

**Comment by ruizajtruss** [comment]
> I'm doing the merge locally too to see if any tests break. So if this passes locally, you won't need to actually do the merge and I'll give my review. 🥳

I just merged 😆

File: swagger-def/definitions/MTOShipment.yaml [hunk 1]
```diff
@@ -51,6 +51,10 @@ properties:
     x-nullable: true
     format: date
     type: string
+  actualDeliveryDate:
+    x-nullable: true
+    format: date
+    type: string
```
Comment (rogeruiz) [inline swagger-def/definitions/MTOShipment.yaml:57]:
```suggestion
  actualDeliveryDate:
    x-nullable: true
    description: The actual date that the shipment was delivered to the destination address by the Prime
    format: date
    type: string
```

**Review by rogeruiz — CHANGES_REQUESTED** [review]
❌ This PR needs changes ...

Great work on this @ruizajtruss. I'm requesting changes to add a description from the comment in the database here to make it easier for folks reading the API documentation to understand what this property is for.

[➡️ Source for the comment on the database column.](https://github.com/transcom/mymove/blob/main/migrations/app/schema/20220901162926_add_scheduled_actual_delivery_dates.up.sql)

**Comment by rogeruiz** [comment]
Just updating here that tests passed locally. So besides the small addition of a
description this is getting closer to getting merged in. 🎉

File: swagger-def/definitions/MTOShipment.yaml (hunk 1 above)
Comment (ruizajtruss) [reply swagger-def/definitions/MTOShipment.yaml]: Added! 🎉

**Review by rogeruiz — APPROVED** [review]
lgtm 🌈

**Pull request merged**